Source/SwiftLintFrameworkTests/RuleTests.swift

```diff
@@ -58,7 +58,7 @@ class RuleTests: XCTestCase {
         XCTAssertFalse([RuleMock1(), RuleMock2()] == [RuleMock1()])
     }
 
-    func testViolationLevelRuleInitsWithConfigDictionary() {
+    func testSecurityLevelRuleInitsWithConfigDictionary() {
```

Comment:
This should be `testSeverityLevelRuleInitsWithConfigDictionary`. All the other tests changed in this file now also say "Security" where they should say "Severity".